generator-angularjs/package.json

```diff
@@ -71,6 +71,7 @@
     "angular-gettext": "2.4.2",
     "angular-messages": "1.3.18",
     "angular-sanitize": "bonitasoft/bower-angular-sanitize#v1.3.21",
+    "angular-ui-bootstrap": "0.14.3",
```

Comment:
Can it creates conflict with existing custom widgets using angular-bootstrap ?

Reply:
good point ! I will try to do it differently

Reply:
I used ngDialog instead